packages/victory-core/src/victory-util/data.js

```diff
@@ -97,8 +97,12 @@ function cleanData(dataset, props) {
 function formatDataFromDomain(dataset, domain, defaultBaseline) {
   const exists = (val) => val !== undefined;
 
-  const [minDomainX, maxDomainX] = domain.x;
-  const [minDomainY, maxDomainY] = domain.y;
+  let [minDomainX, maxDomainX] = domain.x;
```

Comment:
I think it would be better to get all of these explicitly like `const minDomainX = Collection.getMinValue(domain.x)` so that dates are accounted for.